scalableminds/webknossos
# Project-local build scripts + Newrelic integration

Description of changes:
- Before, all build scripts that Jenkins used were in `saltyscm`. Now, they are here in this repo
- Integrates Newrelic agent for backend. License key is stored in Jenkins Environment variable (not sure if there's a better way to store secrets like that)

![bildschirmfoto 2016-08-19 um 18 25 31](https://cloud.githubusercontent.com/assets/335438/17816693/d3a28d8c-663a-11e6-9837-b826f6baaae8.png)

Steps to test:
- Submit a build to Jenkins and check that all packages are created
- Install on dev, check that service is running, check log files, check newrelic config, check nginx config, check init script
- Installation on prod we'll check when we deploy it there first xD

Issues:
- fixes #1430

---


<a href="https://timer.scm.io/repos/537237ce1a00001a003c4a14/issues/1451/create?referer=github" target="_blank">Log Time</a>
